docker-compose.presentation.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.8'
```

Comment:
question: @nicki182: Could you let us know if the file name is in line with what you were thinking?

question: @Jolanto: Are we loading the frontend as well?